boringtun/src/noise/mod.rs

```diff
@@ -336,16 +448,13 @@ impl Tunn {
             remote_idx = p.sender_idx
         );
 
-        let session = {
-            let mut handshake = self.handshake.lock();
-            handshake.receive_handshake_response(p)?
-        };
+        let session = { self.handshake.receive_handshake_response(p)? };
```

Comment:
Do we still need the `{}` brackets surrounding the expression here?

Reply:
No, this is a mistake, I meant to remove this.

Reply:
Done.